raiden/transfer/mediated_transfer/mediator.py

```diff
@@ -35,7 +35,7 @@
     NODE_NETWORK_UNREACHABLE,
     NettingChannelState,
     RouteState,
-    message_identifier_from_prng,
+    message_identifier_from_prng, NODE_NETWORK_UNKNOWN,
```

Comment:
I'd move NODE_NETWORK_UNKNOWN to line 36 and add a new line, for formatting reasons.

Reply:
Done